web/src/beta/features/Modals/ChooseAssetModal/index.tsx

```diff
@@ -185,7 +199,7 @@ const SearchButton = styled(Button)`
 const AssetListWrapper = styled.div`
   display: flex;
   flex-direction: column;
-  overflow-y: scroll;
+  overflow-y: auto;
 `;
```

Comment:
Shall we set a height or max-height on this wrapper? Otherwise it will be very tall if there's a lot of assets.